test/integration/http_integration.cc

```diff
@@ -29,7 +29,7 @@
 #include "test/integration/autonomous_upstream.h"
 #include "test/integration/test_host_predicate_config.h"
 #include "test/integration/utility.h"
-#include "test/mocks/upstream/mocks.h"
+#include "test/mocks/upstream/cluster_info.h"
```

Comment:
was this change reflected in the build rule? i see no change

Reply:
I only changed build rule for one of its deps: integration_lib. Originally it include upstream/mocks.h from this target but i forget to fix it in this PR. So, it's a transitive dep again.